lore/end_users/status/README.md

```diff
@@ -40,3 +46,7 @@ The Zuul Pipelines counter provides an easy look at how many pipelines exist in
 ### Service Status
 
 Nodepool, Zookeeper, and Zuul are three core services that are essential to the stability of a BonnyCI cluster. If anything except the green "OK" message is displayed for these service statuses, then BonnyCI could be experiencing a full service outage.
+
+## Build Status
+
+The [Build Status page](https://travis-ci.org/BonnyCI) links to the Travis-CI page with our repositories. Yes we see the humor in a CI tool using a different CI tool for testing. This is a temporary home for us.
```

Comment:
@rattboi: I was purposefully leaving this section as `## Build Status` so that when we fully switch over to BonnyCI, we could keep the same section and just rewrite it. That way, if we link here, the links won't be broken. For the record, [bonnyci.org](http://bonnyci.org) does not include any build status badges...but we should add those to BonnyCI at some point. I like pictures.

@j2sol: I disagree. We need to have a link to our build status page for our repos, whether that is a Travis-CI page, or BonnyCI doesn't matter.